releasedir/provider.go

```diff
@@ -103,6 +103,8 @@ func (p Provider) newBlobstore(dirPath string) boshblob.DigestBlobstore {
 		blobstore = boshblob.NewLocalBlobstore(p.fs, p.uuidGen, options)
 	case "s3":
 		blobstore = NewS3Blobstore(p.fs, p.uuidGen, options)
```

Comment:
interface{} would look something like this in each of the ExternalBlobstore methods because of a lack of a common interface. Thoughts?

```
func (b ExternalBlobstore) Get(blobID string) (string, error) {
	client, err := b.clientCreator() // (interface{}, error)
	if err != nil {
		return "", err
	}

	file, err := b.fs.TempFile("bosh-gcs-blob")
	if err != nil {
		return "", bosherr.WrapError(err, "Creating destination file")
	}
	defer file.Close()

	switch client := client.(type) {
	case *gcsclient.GCSBlobstore:
		err = client.Get(blobID, file)
	case *s3client.S3Blobstore:
		err = client.Get(blobID, file)
	default:
		return "", bosherr.Error("Unknown blobstore client")
	}
	return file.Name(), err
}
```